test/sql/aggregate/aggregates/test_avg.test

```diff
@@ -56,6 +56,18 @@ SELECT AVG(i) FROM integers WHERE i > 100
 ----
 NULL
 
+statement ok
```

Comment:
Maybe some tests that involve days and months too? And some that are positive and negative?